apps/skilavottord/ws/src/app/modules/recyclingRequest/recyclingRequest.resolver.ts

```diff
@@ -33,7 +34,6 @@ export class RecyclingRequestResolver {
     return res
   }
 
-  @Authorize({ roles: [Role.developer, Role.recyclingFund] })
```

Comment:
This would expose every single recycling requests regardless of if a user should be able to view it or not.

Would rather create a new query that exposes a single `RecyclingRequestModel` and verifies that the user owns the car.

Reply:
Finished to implement a new query (in resolver and service, not frontend ). But I'm wondering if we should put a check on old query: whether user owns the vehicle, return only last recyclingRequest, else return all. If we do this way, we could skip new query and we dont have to change codes in frontend.  @darrikonn what do you think?

Reply:
Makes more sense to have another query IMO